style.css

```diff
@@ -33,30 +46,55 @@ a
 	color: black;
 }
 
-.footerDiv {
-	position: fixed;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	padding-right: 20;
-	background-color: #efefef;
-	text-align: right;
+button
```

Comment:
Any particular reason why we're styling against raw tags here? Not saying there's anything wrong with it, just asking for a bit of context. :)